lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -394,28 +395,26 @@ export class EvaluationClientWrapper {
     }
   }
 
-  public determineAreRequiredCredentialsPresent(matchSubmissionRequirements: SubmissionRequirementMatch[]): Status {
-    let status = Status.INFO;
+  public determineAreRequiredCredentialsPresent(
+    matchSubmissionRequirements: SubmissionRequirementMatch[] | undefined,
+    status: Status
```

Comment:
I think it is not required to be  passed. We are not setting/ we should not set the value passed by reference.